turbinia/api/client/turbinia_api_client/cli/helpers/auth_helper.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Turbinia API client / management tool."""
+
+import logging
+import os
+import sys
+
+from google_auth_oauthlib import flow
+from google.oauth2.credentials import Credentials
+from google.auth.transport.requests import Request
+from google.auth import exceptions as google_exceptions
+
+_LOGGER_FORMAT = '%(asctime)s %(levelname)s %(name)s - %(message)s'
+logging.basicConfig(format=_LOGGER_FORMAT)
+log = logging.getLogger('turbiniamgmt:helpers:auth')
+log.setLevel(logging.INFO)
```

Comment:
Makes sense. I've updated this so that the logging level is only set in the main turbiniamgmt.py file. Default is DEBUG, which is not overly verbose anyway. Let me know if you think we need to change this further.